### DISCUSSION TITLE
Interactive terminals inside `turbo`

### DISCUSSION THREAD

**Discussion by nikero41** [vercel/turborepo#6854] [Help] [answered]
In cases where you need to give input to a workspace task, like pushing to db for example it may ask you to select an option, or some servers have the option to restart if pressing `r`.
Is there a way to do that when running tasks with turbo?

**Comment by anthonyshew** [comment, marked as answer]
@mehulkar is working on this right now! Feel free to add a 👍 reaction to #1235 if you haven't already.
